custom_components/bmw_connected_drive/sensor.py

```diff
@@ -66,21 +120,44 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
                     account, vehicle, attribute_name, attribute_info
                 )
                 devices.append(device)
+        for service in vehicle.state.attributes:
+            # Add sensors for LastTrip, AllTrips & ChargingProfile
+            if service in ('LAST_TRIP', 'ALL_TRIPS', 'CHARGING_PROFILE'):
```

Comment:
Make these CONSTANTS and get them from `const.py` as these are used more often

Reply:
Fixed that :D